src/indexer.ts

```diff
@@ -93,6 +94,7 @@ export class SystemVerilogIndexer {
      * find_files
      */
     public async find_files(token: CancellationToken): Promise<Uri[]> {
+        // eslint-disable-next-line no-async-promise-executor
         return new Promise(async (resolve) => {
```

Comment:
I think the `async` should be removed from this function call instead of being waived.

Reply:
When I remove this `async` (and the corresponding `await` below) some indexer tests fail. Since I don't know the exact intent or usage model, I left it as is.